### ISSUE TITLE
Token list controls

### ISSUE THREAD

**Issue by zspitz** [zspitz/ANTLR4ParseTreeVisualizer#14]
- [ ] Checkbox -- show/hide whitespace tokens
- [ ] Checkbox -- show/hide error tokens

**Comment by bclothier** [comment]
Note that whitespace tokens are implementation defined, depending on the grammar. In our case, [we consider `_` to be whitespace, too](https://github.com/rubberduck-vba/Rubberduck/blob/next/Rubberduck.Parsing/Grammar/VBAParser.g4#L967):

```whiteSpace : (WS | LINE_CONTINUATION)+;```

From [the lexer](https://github.com/rubberduck-vba/Rubberduck/blob/next/Rubberduck.Parsing/Grammar/VBALexer.g4#L309)
```
LINE_CONTINUATION : [ \t]+ UNDERSCORE [ \t]* '\r'? '\n' WS_NOT_FOLLOWED_BY_LINE_CONTINUATION*;
```

[and this, too](https://github.com/rubberduck-vba/Rubberduck/blob/next/Rubberduck.Parsing/Grammar/VBALexer.g4#L306)
```
WS : [ \t];
```

To keep this generic, you might want to consider having a filter list to show/hide certain tokens instead.

**Comment by zspitz** [comment]
@bclothier Agreed, I've edited this to include a filterable list of token types.

However, my intention of whitespace is something we can extract from the token itself -- is the token composed of .NET whitespace characters or not? There may be less value in filtering this way, because as you noted the boundaries of what a given language lexer/parser considers whitespace don't always line up with what .NET considers whitespace. But I still think there is some value.
